.github/workflows/deploy-lambdas.yml

```diff
@@ -0,0 +1,99 @@
+name: Deploy lambdas to S3 and ECR
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - '.github/workflows/deploy-lambdas.yml'
+      - 'lambdas/**'
+
+jobs:
+  deploy-lambda-s3:
+    strategy:
+      matrix:
+        path:
+          - access_counts
+          - es/indexer
+          - pkgevents
+          - pkgpush
+          - pkgselect
+          - preview
+          - s3hash
+          - s3select
+          - status_reports
+          - tabular_preview
+          - transcode
+    runs-on: ubuntu-latest
+    # These permissions are needed to interact with GitHub's OIDC Token endpoint.
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build zip
+        run: |
+          BUILDER_IMAGE=quiltdata/lambda:build-3.8
```

Comment:
would be nice if python version become variable so we could use different versions if we need

Reply:
TBH, I don't even know where this image comes from. And we probably should use different images, not different tags of the same image, for different Python versions.

Reply:
> And we probably should use different images, not different tags of the same image, for different Python versions.

why? e.g. python images uses tags i.e. python:3.8, python:3.9 etc

but anyway nothing change here, I'll take care about that myself

Reply:
> TBH, I don't even know where this image comes from

https://github.com/quiltdata/lambda